client/src/containers/Courses/ViewLessons.js

```diff
@@ -11,7 +11,7 @@ class ViewLessons extends Component {
   }
 
   componentWillReceiveProps(newProps) {
-    getLessons(newProps.courseId).then(res => {
+    getCourseById(newProps.courseId).then(res => {
```

Comment:
This component is ViewLessons, are you sure you need to use getCourseById? You need to load the lessons, don't you?

Reply:
I try this one and it is working. I changed some things in the courses page can you look because it was working my local machine all of these codes

Reply:
Maybe it works, but I think the naming is incorrect. Here it looks like you call getCourseById and finally you get lessons? It doesn't make sense, if you call getCourseById, I would expect to receive a course and not a lesson.

Reply:
ok make sense